src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyService.kt

```diff
@@ -77,4 +79,32 @@ class LicencePolicyService(private val policies: List<LicencePolicy> = listOf(PO
       .allAdditionalConditions()
       .find { it.code == conditionCode }
       ?: error("Condition with code: '$conditionCode' and version: '$version' not found.")
+
+  fun getCurrentStandardApConditions(licenceType: LicenceType): List<StandardCondition> {
+    return if (licenceType == LicenceType.PSS) {
+      emptyList()
+    } else {
+      currentPolicy().standardConditions.standardConditionsAp.mapIndexed { index, standardConditionAp ->
+        StandardCondition(
+          code = standardConditionAp.code,
+          sequence = index,
+          text = standardConditionAp.text,
+        )
+      }
+    }
+  }
+
+  fun getCurrentStandardPssConditions(licenceType: LicenceType): List<StandardCondition> {
+    return if (licenceType == LicenceType.AP) {
+      emptyList()
+    } else {
+      currentPolicy().standardConditions.standardConditionsPss.mapIndexed { index, standardConditionPss ->
+        StandardCondition(
+          code = standardConditionPss.code,
+          sequence = index,
+          text = standardConditionPss.text,
+        )
```

Comment:
once we're happy this works, we can have a transformer from the policy model to the entity